tests/unit/data-viewer-cell.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { shallowMount } from '@vue/test-utils';
+import DataViewerCell from '@/components/DataViewerCell.vue';
+
+describe('Data Viewer Cell', () => {
+  it('should instantiate with no value', () => {
+    const wrapper = shallowMount(DataViewerCell);
+
+    expect(wrapper.exists()).toBeTruthy();
+  });
+
+  it('should display the value', () => {
+    const wrapper = shallowMount(DataViewerCell, {
+      context: {
+        props: {
+          value: 'my_value',
+        },
+      },
+    });
+
+    expect(wrapper.text()).toEqual('my_value');
+  });
+
+  it('should have specific call when selected', () => {
+    const wrapper = shallowMount(DataViewerCell, {
+      context: {
+        props: {
+          value: 'my_value',
+          isSelected: true,
+        },
+      },
+    });
+
+    expect(wrapper.classes()).toContain('data-viewer-cell--active');
+  });
```

Comment:
I would add a test for undefined value and for some other value types, so it's clearly specified how they are stringified.